external/libp2p-service.js

```diff
@@ -9,11 +9,14 @@ const pipe = require('it-pipe');
 const {sha256} = require('multiformats/hashes/sha2');
 const {v1: uuidv1} = require("uuid");
 const PeerId = require("peer-id");
-const fs = require("fs");
+const fs = require('fs');
 const {time} = require("streaming-iterables");
 const { BufferList } = require('bl')
 const constants = require('../modules/constants');
 
+const configFilename = process.argv.length === 3 && process.env.NODE_ENV === 'development'
```

Comment:
This should be determined in ot-node, not external service

Reply:
Done in [latest commit](https://github.com/OriginTrail/ot-node/pull/1698/commits/8901b8da1462a2047e8df62f1d9e86153edc83eb)